src/frontend/src/styles/pages/Dashboard.scss

```diff
@@ -2,26 +2,35 @@
 @use '@carbon/react/scss/breakpoint' as *;
 @use '../mixins' as *;
 
-.dashboard-page__banner,
-.dashboard-page__description {
-    margin-left: -20px;
-    margin-right: -20px;
-    @include breakpoint-down(md) {
-        margin-left: 0;
-        margin-right: 0;
-    }
+.dashboard-page{
+    min-height: 70vh;
+    padding: $spacing-08;
+    background: linear-gradient(.60turn, #0f4beb,transparent);
```

Comment:
this is great, and I was thinking something similar. But think bigger - let's get this in more places, on more pages, and keep it consistent. I've already introduced something that can set this pattern up in a theme

https://github.com/albert118/Studio-Management-System/pull/106/files#diff-8cd8865459732fa8f273ace6c05385879c19c0aa4e8aecdc3312f9dacd9343e7R30-R36

Reply:
It'll be easier to follow up in a second PR to do all the backgrounds though, 😄

Reply:
Yep good idea, personally think the home page should look a little different to the other pages as those a more for function. That said I think we can leave that for next semester.